lib/Doctrine/ORM/EntityManagerInterface.php

```diff
@@ -320,7 +320,8 @@ public function hasFilters();
     /**
      * {@inheritDoc}
      *
-     * @psalm-param string|class-string<T> $className
+     * @param string $className
+     * @psalm-param class-string<T> $className
```

Comment:
This is wrong, `string` is also accepted here: https://github.com/doctrine/persistence/blob/79e97bcb1dfe3b242961253d81b8357a04e96ca1/lib/Doctrine/Persistence/Mapping/AbstractClassMetadataFactory.php#L226

Reply:
Indeed, so it's a phpstan issue.

Reply:
Probably yes. Note that I don't know what that `:` notation is useful for, maybe we should deprecate it if it's just an alternate way to do things…